### PR TITLE
env config done

### PR DESCRIPTION
**Pull request by VenomFate-619**
`.env.production` early not added in ignore file that why it is as it is.
I was not too sure what to add in gitignore as well.
**I had added `.env.example`**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#put this in .env.local
+NEXT_PUBLIC_BASE_API_URL='https://api.realdevsquad.com'
```
Comment (swarajpure) [inline .env.example:2]: What's `.env.example`?

File: .gitignore [hunk 2]
```diff
@@ -27,6 +27,8 @@ yarn-error.log*
 
 # local env files
 .env.local
+.env.test
+.env.development
```
Comment (swarajpure) [inline .gitignore:31]: `.env.development` and `.env.test` don't need to be in gitignore

**Review by swarajpure — CHANGES_REQUESTED** [review]
Thanks for the PR!

What's `.env.example`?

File: .env.example (hunk 1 above)
Comment (VenomFate-619) [reply .env.example:2]: To give an idea to a new contributor to set up the repo env file, as I had ignored the .env.* files,

File: .gitignore (hunk 2 above)
Comment (VenomFate-619) [reply .gitignore:31]: I thought that it will contain, secret keys which are not to be exposed directly, that is why I had added it in git.ignore file

Comment (swarajpure) [reply .gitignore:31]: No, those are not secret keys
Nothing is secret on frontend 😂

File: .env.example (hunk 1 above)
Comment (swarajpure) [reply .env.example:2]: `.env.local` takes higher priority than env specific variables and if someone wants to override the env variables, they should be playing with this file and this one should be in gitignore

**Comment by VenomFate-619** [comment]
@swarajpure Should I close this  PR or any update to de done

**Comment by swarajpure** [comment]
> @swarajpure Should I close this PR or any update to de done

Why close? Do you not want it to be merged? 🤔

**Comment by VenomFate-619** [comment]
> > @swarajpure Should I close this PR or any update to de done
> 
> Why close? Do you not want it to be merged? thinking

Yes, I want it to be merged, but it has been opened for a long time So I thought that it won't be merged in future.

**Comment by swarajpure** [comment]
> > > @swarajpure Should I close this PR or any update to de done
> > 
> > 
> > Why close? Do you not want it to be merged? thinking
> 
> Yes, I want it to be merged, but it has been opened for a long time So I thought that it won't be merged in future.

It's important that the reviewer is in loop when you make changes, otherwise they won't come to know that required changes are done and hence PR might become stale. Please ensure to tag the reviewers ***on Discord*** channel once you've made changes (can't rely on GitHub notifications, there are always tons of them)

**Comment by MehulKChaudhari** [comment]
Closing this PR, we already have `.env.development` and `.env.production`. And I don't think so we need `.env.test` 

But if you guys think we want this PR we can always discuss and open this PR.

**Pull request closed without merging**